java/build.sh

```diff
@@ -3,12 +3,24 @@ GROUP_ID="com.nvidia.cuvs"
 SO_FILE_PATH="./internal"
 
 if [ -z "$CMAKE_PREFIX_PATH" ]; then
-    export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
+  export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
 fi
 
 cd internal && cmake . && cmake --build . \
-  && cd .. \
-  && mvn install:install-file -DgroupId=$GROUP_ID -DartifactId=cuvs-java-internal -Dversion=$VERSION -Dpackaging=so -Dfile=$SO_FILE_PATH/libcuvs_java.so \
+  && cd ..
+
+# Generate Panama FFM API bindings and update (if any of them changed)
+/bin/bash panama-bindings/generate-bindings.sh
+
+BINDINGS_GENERATION_RETURN_VALUE=$?
+if [ $BINDINGS_GENERATION_RETURN_VALUE != 0 ]
+then
+  echo "Bindings generation did not complete normally (returned value ${BINDINGS_GENERATION_RETURN_VALUE})"
+  echo "Forcing this build process to abort"
+  exit 1
+fi
+
```

Comment:
```suggestion
```

Instead of having this custom return-code handling, would you please consider:

1. adding `#!/bin/bash` to the first line
2. adding `set -e -u -o pipefail` somewhere after that, near the top, to ensure any command exiting with a non-0 exit code causes the entire script to exit

That's simplify the code here a bit.